.github/workflows/quick-jobs.yml

```diff
@@ -124,4 +124,36 @@ jobs:
       - uses: actions/checkout@v4
       - name: Are buildinfo docs up to date?
         run: make doc/buildinfo-fields-reference.rst
+  release-project:
+    name: Check Release Project
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set PATH
+        run: |
+          echo "$HOME/.cabal/bin" >> $GITHUB_PATH
+      - uses: actions/cache@v3
+        with:
+          path: ~/.cabal/store
+          key: linux-store-release-project-check
+      # See https://github.com/haskell/cabal/pull/8739
+      - name: Sudo chmod to permit ghcup to update its cache
+        run: |
+          if [[ "${{ runner.os }}" == "Linux" ]]; then
+            sudo ls -lah /usr/local/.ghcup/cache
+            sudo mkdir -p /usr/local/.ghcup/cache
+            sudo ls -lah /usr/local/.ghcup/cache
+            sudo chown -R $USER /usr/local/.ghcup
+            sudo chmod -R 777 /usr/local/.ghcup
+          fi
```

Comment:
Is this workaround for ghcup still needed?  The original issue is closed:
- https://github.com/actions/runner-images/issues/7061
- cf. https://github.com/herp-inc/hs-opentelemetry/commit/ef402ff9b641e74fcf86adaf38bd5ce8e224fc68

Reply:
Is there an example in our workflows that doesn't use this workaround (all the ones in `quick-jobs.yml` seem to)? Should we raise a separate pull request to strip these workarounds out altogether?

Reply:
I'd prefer this be dinner in the current PR, personally. Too granular PRs are straining our reviewing resource. And I don't feel excited about approving blind copy-paste that is even not needed anymore most likely.

Reply:
Those `sudo ...` commands are annoyingly repetitive but this is used with every other job in `quick-jobs.yml`. I saw the pattern and tried to fit in with it.

Reply:
I've remove the extra duplicate of the workaround that would have been added by this new job but I've not done the others in this workflow, nor those in other workflows.

Reply:
Sounds good, thank you!